frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -666,6 +668,49 @@ export default function SurveyEdit(): JSX.Element {
+                                                                Device Types
+                                                                <LemonSelect
+                                                                    value={
+                                                                        value?.deviceTypesMatchType ||
+                                                                        SurveyMatchType.Contains
+                                                                    }
+                                                                    onChange={(matchTypeVal) => {
+                                                                        onChange({
+                                                                            ...value,
+                                                                            deviceTypesMatchType: matchTypeVal,
+                                                                        })
+                                                                    }}
+                                                                    data-attr="survey-device-types-matching-type"
+                                                                    options={Object.keys(SurveyMatchTypeLabels).map(
+                                                                        (key) => ({
+                                                                            label: SurveyMatchTypeLabels[key],
+                                                                            value: key,
+                                                                        })
+                                                                    )}
+                                                                />
+                                                                <PropertyValue
+                                                                    propertyKey="$device_type"
+                                                                    type={PropertyFilterType.Person}
+                                                                    onSet={(deviceTypes: string[]) =>
+                                                                        onChange({
+                                                                            ...value,
+                                                                            deviceTypes: deviceTypes,
+                                                                        })
+                                                                    }
+                                                                    operator={PropertyOperator.Exact}
+                                                                    placeholder="Enter value..."
+                                                                    autoFocus={true}
+                                                                    addRelativeDateTimeOptions={true}
```

Comment:
can this be ignored? does not feel like we need this

Reply:
operator is mandatory, the other three I've removed